h/feeds/rss.py

```diff
@@ -9,18 +13,16 @@
 _ = i18n.TranslationStringFactory(__package__)
 
 
-def _pubDate_string_from_annotation(annotation):
-    """Return a correctly-formatted pubDate string for the given annotation.
+def _pubDate_string(timestamp):
+    """Return a RFC822-formatted pubDate string for the given timestamp.
```

Comment:
Ah! The feed validator refers to the older RFC 822 which got superseded by RFC2822. The date format has stayed the same. I will update the comment to say RFC 2822, that's also what the Python docs for `email.utils.formatdate` say.